File: examples-sdk/python/507-villog-bench-file-drain.py
#!/usr/bin/env python3
"""507-villog-bench-file-drain — Python SDK equivalent
Compile: vil compile --from python --input 507-villog-bench-file-drain.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/507-villog-bench-file-drain.ts
#!/usr/bin/env tsx
// 507-villog-bench-file-drain — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 507-villog-bench-file-drain.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
